overhauled the-3D-panel.md (#51)
iXtenda/iVP-gitbook: home/ivp-planning/user-interface/the-3d-panel.md

---
description: >-

>   The 3D panel primarily serves as a visualization area. Similar to the 2D panel, it allows you to position, adjust size, and rotate machinery and objects as if you were working with a traditional paper floor plan.

Reviewer: The 3D panel is not a visualization area. It allows planning just like the 2D panel.

---

# 3D-Panel

The 3D-Panel acts as a visualization area where you can work in a three dimensional representation of the room or factory. It is possible to [place](../machines/first-steps-with-3d-object.md), [move](../machines/selecting-and-moving-objects.md), [scale and rotate](../machines/scale-and-rotate-objects.md) machines and objects like in the 2D-panel. However, the 3D-panel also offers functions to take [screenshots](../advanced-tools/creating-screenshots.md) or to work with [virtual cameras](virtual-camera-panel.md) to [record videos](../advanced-tools/virtual-cameras.md).

> 

Reviewer: The first sentence is redundant with the description above. Taking screenshots and working with virtual cameras is also possible in the 2D panel. Do we really need to mention these features here? This just makes the documentation get outdated faster. I think this paragraph and the description above could be heavily trimmed down.

> The 3D-Panel allows you to plan within a three-dimensional representation of the room or factory. You can [place](../machines/first-steps-with-3d-object.md), [move](../machines/selecting-and-moving-objects.md), [scale and rotate](../machines/scale-and-rotate-objects.md) machines and objects like in the 2D-panel. This streamlined functionality enures efficient and flexible planning.

Reviewer: There is no need to mention the 2D panel over and over again.

![](../../../.gitbook/assets/UI\_3D\_panel.png)

## The toolbar of the 3D-Panel

The toolbar is located at the top of the panel and contains several buttons that give you quick access to 3D-specific functions and settings.

|                                                                                    |                                                                                                                                                                                              |
| ---------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 

>  ![](../../../.gitbook/assets/iVP\_icon\_3D\_toggle_camera_mode.png)                                | Toggles the camera mode

Reviewer: Needs more explanation. What does the camera mode do?

>  ![](../../../.gitbook/assets/iVP\_icon\_3D\_toggle_camera_mode.png)                                | Toggles the camera mode, which simulates a traversal between the positions of your placed cameras

Reviewer: This still does not explain what the different camera modes actually do. The transition does not need to be mentioned here.

| ![](../../../.gitbook/assets/iVP_icon_3D_selected_view.jpg)              | Allows you to switch between different [virtual cameras](virtual-camera-panel.md)                                                                                                            |
| ![](../../../.gitbook/assets/iVP_icon_3D_move.png)                              | Switches to a mode that makes it possible to [move](../machines/selecting-and-moving-objects.md) the selected object(s) by using the mouse|
| ![](../../../.gitbook/assets/iVP_icon_3D_rotate.png)                      | Switches to a mode that makes it possible to [rotate](../machines/scale-and-rotate-objects.md#rotate-objects) the selected object(s)|
| ![](../../../.gitbook/assets/iVP\_icon\_3D\_scale.png)                             | Switches to a mode that makes it possible to [scale](../machines/scale-and-rotate-objects.md#scale-objects) the selected object(s)|

> | ![](../../../.gitbook/assets/iVP_icon_3D_global_space.png)          |  "Global space" refers to a coordinate system that isn't tied to any particular object. Its axes remain fixed in direction, regardless of any changes made to an object                                                                                                                                                                                         |

Reviewer: The explanation here is good 👍 We should add that this toggles between global and local space.

> |  ![](../../../.gitbook/assets/iVP_icon_3d_field_of_view.png)          |  Lets you adjust the Field of View in the 3D-View

Reviewer: This is more precise. We should also stay consistent with calling it a _panel_ instead of a _view_. Please make sure this is the case everywhere on this page.

```suggestion
|  ![](../../../.gitbook/assets/iVP_icon_3d_field_of_view.png)          |  Lets you adjust the Field of View of the 3D panel's camera.
```

> | ![](../../../.gitbook/assets/iVP\_icon\_3D\_camera_reset_view.png)          |  Restores the 3D-View to its initial default settings, akin to when you first launch the program

Reviewer: ```suggestion
| ![](../../../.gitbook/assets/iVP\_icon\_3D\_camera_reset_view.png)          |  Restores the 3D panel's camera to its initial position
```

> | ![](../../../.gitbook/assets/iVP_icon_3D_global_space.png)          |  "Global space" refers to a coordinate system that isn't tied to any particular object. Its axes remain fixed in direction, regardless of any changes made to an object. This setting can be toggled between global and local space                                                                                                                                                                                         |

Reviewer: You are still not explaining what the toggle actually does, e.g. it is unclear that this affects the 3D gizmo (keep in mind that our users won't know what a gizmo is). Just generally talking about a "setting" here is wrong (this is a toggle) and confusing.

| ![](../../../.gitbook/assets/iVP\_icon\_3D\_add_camera.png) | Adds a new [virtual camera](../advanced-tools/virtual-cameras.md)                                                                                                                            |

> | ![](../../../.gitbook/assets/iVP\_icon\_3D\_update_camera.png)          | Adjusts the rotation and position of the selected camera to match the position of the 3D view.

Reviewer: ```suggestion
| ![](../../../.gitbook/assets/iVP\_icon\_3D\_update_camera.png)          | Adjusts the position and rotation of the selected virtual camera to match the position of the 3D panel's camera.
```

| ![](../../../.gitbook/assets/iVP_icon_3D_aspect-ratio.jpg)                     | Opens a drop-down menu, from which you can change the aspect ratio of the 3D-View (e.g. 4:3, 21:9)                                                                                                                                     |
| ![](../../../.gitbook/assets/iVP\_icon\_3D\_screenshot.png)                        | Takes a [screenshot](../advanced-tools/creating-screenshots.md) of the 3D-view                                                                                                                                        |

> | ![](../../../.gitbook/assets/iVP\_icon\_3D\_skybox.png)          | Allows customization of your plan-file's skybox by incorporating either a Cubic or Panoramic file

Reviewer: ```suggestion
| ![](../../../.gitbook/assets/iVP\_icon\_3D\_skybox.png)          | Allows customization of your plan-file's skybox by incorporating either a cubic or panoramic image
```
